lnxcontroller/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(HEADERS)
+set(CPPS lnxcontroller.cpp)
+
+add_library(lnxcontroller STATIC ${HEADERS} ${CPPS})
```

Comment:
Unless I miss something this lib is not linked to anything. `lnxcontroller.cpp` is already compiled as part of the `linux` library